README.md

```diff
@@ -135,28 +135,6 @@ ark restore describe <RESTORE_NAME>
 
 For more information, see [the debugging information][18].
 
-### Clean up
```

Comment:
maybe leave this section, and specify that you can just delete the NS, and note that backups/snapshots will **not** be deleted

Reply:
Ok. I'll make it clearer that this is optional and only if you want to remove stuff.